distribution/alliance/pom.xml

```diff
@@ -268,7 +268,7 @@
                                 </descriptor>
                                 <!-- Alliance Apps -->
                                 <descriptor>
-                                    file:${basedir}/target/classes/alliance-features.xml
+                                    mvn:${project.groupId}/${project.artifactId}/${project.version}/xml/features
```

Comment:
❗️ Same comment/question as https://github.com/codice/ddf/pull/2241#discussion_r135345168.

Reply:
👍 Fixed in https://github.com/codice/alliance/pull/367.